src/frontend/web_application/src/services/i18n.js

```diff
@@ -0,0 +1,73 @@
+import { provideTranslate, negociateLocale, createTranslator } from '@gandi/react-translate';
+
+const availableTranslations = {
+  /* eslint-disable global-require */
+  en: require('../../locales/en/main.json'),
+  fr: require('../../locales/fr/main.json'),
+  /* eslint-enable global-require */
+};
+const defaultLocale = 'en';
+
+export function changeLocale(translator, locale) {
+  translator.registerTranslations(locale, availableTranslations[locale]);
+  translator.setLocale(locale);
+}
+
+export function getLocale() {
+  let preferedUserLocales = [];
+
+  if (BUILD_TARGET === 'browser') {
+    preferedUserLocales = [
+      document.cookie.locale,
+      ...window.navigator.languages,
+      (window.navigator.language || window.navigator.userLanguage),
+    ].filter(rawLocale => rawLocale);
```

Comment:
.filter() here is unnecessary
https://github.com/Gandi/react-translate/blob/master/src/tools/negociateLocale.js#L12